packages/ripple-address-codec/HISTORY.md

```diff
@@ -4,7 +4,8 @@
 ### Breaking Changes
 * Bump typescript to 5.x
 * Remove Node 14 support
-* Remove `assert` dependency. If you were catching `AssertionError` you need to change to `Error`.
+* Remove `create-hash` in favor of `@noble/hashes`
+* Remove `assert` dependency. If you were catching `AssertionError` you need to change to `Error`
```

Comment:
TODO: ByteArray changes

Reply:
Is that really a change though?  I believe we just typed it.

Reply:
ah, I see, this is breaking changes